### ISSUE TITLE
Show message about including `.load!` after config

### ISSUE THREAD

**Issue by cllns** [lotus/model#280]
I got this following error, even though I had set both `adapter` and `mapping`
```
#<Lotus::Model::Adapters::NoAdapterError: Cannot invoke `create' on repository. Please check if `adapter' and `mapping' are set.>
```

The problem was that the end of the config block didn't call `.load!`. 

It'd be nice if the error handling checked to see if that was the problem, and provided a specific error message in that case!

**Comment by AlfonsoUceda** [comment]
@cllns thanks for the issue, could you make a PR?

**Comment by jodosha** [comment]
@cllns Okay, In our docs we specify a lot of times that `.load!` should be invoked before to use it. See [README](https://github.com/lotus/model/blob/master/README.md). In `lotusrb`, the generated code does it for you.

We're open for an improved error message, unless it requires to change too much the internal code. Thank you very much :smile:

**Comment by cllns** [comment]
Closed via #282 !
